internal/signerverifier/ssh/ssh_test.go

```diff
@@ -0,0 +1,101 @@
+package ssh
+
+import (
+	"context"
+	"path"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/gittuf/gittuf/internal/common"
+	"github.com/stretchr/testify/assert"
+)
+
+// Basic smoke test for ssh package for all supported keys
+func TestSSH(t *testing.T) {
+	testSSHKeys := common.TestSSHKeys(t)
+
+	keyidRSA := "SHA256:ESJezAOo+BsiEpddzRXS6+wtF16FID4NCd+3gj96rFo"
+	keyidECDSA := "SHA256:oNYBImx035m3rl1Sn/+j5DPrlS9+zXn7k3mjNrC5eto"
+	keyidEd25519 := "SHA256:cewFulOIcROWnolPTGEQXG4q7xvLIn3kNTCMqdfoP4E"
+
+	tests := []struct {
+		keyName string
+		keyID   string
+	}{
+		{"rsa", keyidRSA},
+		{"rsa_enc", keyidRSA},
+		{"rsa.pub", keyidRSA},
+		{"ecdsa", keyidECDSA},
+		{"ecdsa_enc", keyidECDSA},
+		{"ecdsa.pub", keyidECDSA},
+		{"ed25519", keyidEd25519},
+		{"ed25519_enc", keyidEd25519},
+		{"ed25519.pub", keyidEd25519},
+	}
+
+	for _, test := range tests {
+		t.Run(test.keyName, func(t *testing.T) {
+			if strings.Contains(test.keyName, "_enc") {
+				if runtime.GOOS == "windows" {
+					t.Skip("TODO: test encrypted keys on windows")
```

Comment:
I just spent way too long trying to fix this for Windows, with no success. Depending on the version of OpenSSH -- not the default, and not the one bundled with git either on the GHA runner -- it does indeed work as expected (e.g. with OpenSSH_for_Windows_8.6p1, LibreSSL 3.3.3 installed via choco). Unfortunately, I still can't test it, because the script defined via SSH_ASKPASS fails with a non-explanatory error message:
```
CreateProcessW failed error:193
ssh_askpass: posix_spawnp: Unknown error
```

I suggest to ticketize this issue and move on. That is:

1. Review & merge this PR
2. Integrate the signer/verifier from this PR with TUF metadata
3. Revisit the Windows issue later.